config/taxonomy/base/base.yaml

```diff
@@ -31,3 +31,36 @@ definitions:
       dataformat:
         $ref: "#/definitions/Dataformat"
     additionalProperties: false
+  Source:
```

Comment:
Shouldn't the taxonomy include enums for the plugin types, dependency types, capability scope, and the supported actions?

Reply:
These enums are defined in the CRD and are not configurable by the user so we decided not include them in the taxonomy